packages/jetbrains-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -3,7 +3,30 @@
     <name>MongoDB</name>
     <vendor email="compass@mongodb.com" url="https://www.mongodb.com/">MongoDB, Inc.</vendor>
     <description><![CDATA[
-    Enjoy working with MongoDB and Java with autocompletion, intelli-sense suggestions based on defined queries and indexes.<br>
+    <p><b>IMPORTANT: THIS PLUGIN IS IN STILL IN DEVELOPMENT</b></p>
+
+    <p>MongoDB's official plugin includes early-stage features that are still being developed and tested. We strongly advise exercising caution when using the plugin in the current state.</p>
+
+    <p>Please report any issues or feature requests.</p>
```

Comment:
Could we add a link around `issues or feature requests` to our GH issues?

Reply:
Yeah that sounds good, doing it right now!